### PR TITLE
Cockpit ReX integration manual

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

**Review by tbrisker — CHANGES_REQUESTED** [review]
Can you please add this also to the released version of the manual (if it's in that version already), and to the 1.24 headline features?

File: plugins/foreman_remote_execution/nightly/index.md [hunk 1]
```diff
@@ -549,6 +549,17 @@ hammer job-invocation create --job-template "Run Command - SSH Default" \
 hammer job-invocation output --id 155 --host rex01.example.com
 ```
 
+## 3.6 Cockpit integration
+
+To automatically sign to managed host's Cockpit web interface, click on "Web
+Console" button on the host detail page. The button only appears when plugin
+was installed with `--enable-foreman-proxy-plugin-remote-execution-cockpit`
+option. Managed host must also have installed package named `cockpit-system`.
```
Comment (tbrisker) [inline plugins/foreman_remote_execution/nightly/index.md]:
```suggestion
option. The managed host must also have the `cockpit-system` package installed.
```

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 2]
```diff
@@ -15,5 +15,6 @@ This section will be updated prior to the next release.
 * Ubuntu Xenial (16.04) is no longer supported. Users should upgrade to Bionic (18.04) prior to upgrading Foreman to this release.
 * Debian Stretch (9) is no longer supported. Users should upgrade to Buster (10) prior to upgrading Foreman to this release.
 * Smart Variables have been removed. Users should use Smart Class Parameters or Global Parameters instead. [Further details](https://community.theforeman.org/t/dropping-smart-variables/16176)
+* Cockpit single sign-on via "Web Console" button. For more information how to set it up, head over to [Remote Execution](/plugins/foreman_remote_execution/1.7/index.html#3.6CockpitIntegration) manual.
```
Comment (tbrisker) [inline _includes/manuals/nightly/1.2_release_notes.md]: Can you move this up to the Headline Features section above please? Also, looks like you forgot to add the change in the 1.7 rex manual

**Comment by lzap** [comment]
Done

**Comment by lzap** [comment]
Wait I've missed something.

**Comment by lzap** [comment]
Ready now.

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+* Cockpit single sign-on via "Web Console" button. For more information how to set it up, head over to [Remote Execution](/plugins/foreman_remote_execution/1.7/index.html#3.6CockpitIntegration) manual.
```
Comment (tbrisker) [inline _includes/manuals/nightly/1.2_release_notes.md]: Oops, i missed that this was the nightly manual, could you place this in the 1.24 manual instead? Sorry for that.

Comment (lzap) [reply _includes/manuals/nightly/1.2_release_notes.md]: Yeah I was wondering why it was so empty :-) Done.

**Review by tbrisker — APPROVED** [review]
Thanks @lzap !

**Pull request merged**